src/series/bar.js

```diff
@@ -26,12 +33,12 @@
                     .width(width)
                     .height(0);
 
+                pathGenerator.height(0);
```

Comment:
I'm really interested to have a proper look at this but it's going to be
tomorrow afternoon at the earliest before I get a chance :(

On Wed, 3 Jun 2015 at 18:12 Colin Eberhardt <notifications@github.com>
wrote:

> In src/series/bar.js
> <https://github.com/ScottLogic/d3-financial-components/pull/399#discussion_r31640217>
> :
>
> > @@ -26,12 +33,12 @@
> >                      .width(width)
> >                      .height(0);
> >
> > +                pathGenerator.height(0);
>
> Probably not - well spotted [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ScottLogic/d3-financial-components/pull/399/files#r31640217>
> .
>